src/app/features/tech-record/components/tech-record-amend-vin/tech-record-amend-vin.component.ts

```diff
@@ -8,17 +8,25 @@ import { CustomFormControl, FormNodeTypes, FormNodeWidth } from '@forms/services
 import { TechRecordModel, VehicleTechRecordModel } from '@models/vehicle-tech-record.model';
 import { Actions, ofType } from '@ngrx/effects';
 import { TechnicalRecordService } from '@services/technical-record/technical-record.service';
-import { updateTechRecordsSuccess } from '@store/technical-records';
-import { take } from 'rxjs';
+import { updateTechRecordsSuccess, updateVin, updateVinSuccess } from '@store/technical-records';
+import { Subject, take, takeUntil } from 'rxjs';
+import { select, Store } from '@ngrx/store';
+import { TechnicalRecordServiceState } from '@store/technical-records/reducers/technical-record-service.reducer';
 
 @Component({
   selector: 'app-change-amend-vin',
   templateUrl: './tech-record-amend-vin.component.html'
 })
-export class AmendVinComponent implements OnInit {
+export class AmendVinComponent implements OnInit, OnDestroy {
+  private destroy$ = new Subject<void>();
   vehicle?: VehicleTechRecordModel;
   techRecord?: TechRecordModel;
 
+  user!: {
```

Comment:
What's this user object?

Reply:
It was a hangover from something else I'd tried, i'll remove it